otx/cli/tools/train.py

```diff
@@ -113,9 +113,9 @@ def parse_args():
         help="Location where trained model will be stored.",
     )
     parser.add_argument(
-        "--save-logs-to",
+        "--output-path",
```

Comment:
Is this an argment that spits out both model weights and log at the same time?

Reply:
In mmXX family, yes. mmXX task saves model weight and log file while model is training.

Reply:
I'm sorry, but I didn't understand that it was a mmXX task. Do you mean the inference task of each task? And why should this be separated from `--model-weight-to`? 
I think `output-path` is a confusing word from the user's point of view.

Reply:
mmXX task means any task using mmcv based training. As f as I know, inference and train tasks of mmcls, mmdet and mmseg save the model weight and log file while task is working.
I agree that user can be confused. but `--model-weight-to` is used for path where OTX final model weight is saved and `--output-path` is used for mmXX output file. So, if user doesn't want to save an intermediate result of training, then user can save it in /tmp , by not setting `--output-path`.
But I think it's also good design choiceto make an intermediate result from mmXX framework saved. how do you think about?

Reply:
Yes I understand your explanation. But I still think `output-path` is not appropriate. Rather confusing.

Reply:
ok. then I'll modify it to `work-dir`.

Reply:
I also agree with Harim's opinion. Although there is a difference between OTX tasks and MM***, why should we divide individually? How about using same path for OTX tasks and MM*** tasks?

AFAIK, Users don't need to check the output of MM***.

Reply:
Actually, I think it's out of scope to decide whether to combine them or not.
how about to discuss about it and make separate PR?